packages/icons/src/Icon.js

```diff
@@ -10,29 +10,35 @@ const Icon = ({ name, title, desc, titleId, descId, ...props }) => {
   ariaLabelledBy += desc && descId ? ` ${descId}` : ''
   ariaLabelledBy = ariaLabelledBy ? ariaLabelledBy : undefined
 
-  return (
-    Component && (
-      <Component
-        title={title}
-        desc={desc}
-        titleId={titleId}
-        descId={descId}
-        aria-hidden={!!ariaLabelledBy}
-        aria-labelledby={ariaLabelledBy}
-        {...props}
-      />
+  if (!Component) {
+    if (process.env.NODE_ENV !== 'production')
+      console.trace(`icon ${iconName} does not exist`)
+    return null
+  } else {
+    return (
+      Component && (
```

Comment:
This check `Component && ` shouldn't be necessary anymore, since we know from the conditional that Component does exist.

Reply:
@jes708 , good 👁 , updated.